### PR TITLE
Improve snowflake sink connector doc

### PR DESCRIPTION
**Pull request by RobertIndie**
### Motivation

Improve the snowflake sink connector doc.

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: docs/snowflake-sink.md [hunk 1]
```diff
@@ -3,105 +3,154 @@ dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-snowflake"
...
-There are two fields in the table: metadata and content. Metadata is ancillary information in content, such as `topic`, `messageId`, `publishTime`, and so on.
-By default, the following metadata fields of Pulsar will be created as the metadata:
+You can get the public key passphrase `(MIIBIjA…)` by running the following command:
+
+```sh
+grep -v "\-\-\-" rsa_key.pub | tr -d '\n'
 ```
-metadataField=__message_id__,__partition__,__topic__,__event_time__
+
+5. Create Snowflake objects and grant permission
```
Comment (shibd) [inline docs/snowflake-sink.md:52]: This step should not belong to `Prerequisites`; it should be a separate step: `### 1. Create Snowflake objects and grant permission`

File: docs/snowflake-sink.md [hunk 2]
```diff
@@ -3,105 +3,154 @@ dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-snowflake"
...
-   pulsar-io-snowflake-{{connector:version}}.nar
-   ```
+![Data Sample](images/data-sample.jpg)
 
-## Work with Function Mesh
+## Configuration Properties
 
-You can pull the Snowflake sink connector Docker image from the [Docker Hub](https://hub.docker.com/r/streamnative/pulsar-io-snowflake) if you use [Function Mesh](https://functionmesh.io/docs/connectors/run-connector) to run the connector.
 
 # How to configure
```
Comment (shibd) [inline docs/snowflake-sink.md:151]: Need remove this line

File: docs/snowflake-sink.md [hunk 3]
```diff
@@ -3,105 +3,151 @@ dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-snowflake"
...
+The prerequisites for connecting a Snowflake sink connector to external systems include:
 
-## Table name mapping
-The Snowflake sink connector supports automatically creating a table when the table does not exist. You can configure the following options:
-```
-autoCreateTable=true
+1. Prepare a snowflake account
+2. Get the account URL from the `Admin - Accounts` page and click the link. It should be the format like `https://<account_identifier>.snowflakecomputing.com`.
+
+![Get the account URL link](images/get-account-url.jpg)
```
Comment (shibd) [inline docs/snowflake-sink.md:21]: Does this image have to be added? Is it possible to simplify it?

Comment (RobertIndie) [reply docs/snowflake-sink.md]: Yes. We can delete it. The statement is already clear.

**Review by shibd — APPROVED** [review]

**Comment by shibd** [comment]
Please help cherry-pick to the history branch: branch-3.0, branch-3.1

**Pull request merged**